supervisor/shared/tick.c

```diff
@@ -102,12 +98,8 @@ void supervisor_tick(void) {
     filesystem_tick();
     #endif
 
-    #ifdef CIRCUITPY_GAMEPAD_TICKS
-    if (!(port_get_raw_ticks(NULL) & CIRCUITPY_GAMEPAD_TICKS)) {
-        #if CIRCUITPY_GAMEPADSHIFT
-        gamepadshift_tick();
-        #endif
-    }
+    #ifdef CIRCUITPY_AUTORELOAD_DELAY_MS
+    autoreload_tick();
     #endif
```

Comment:
`autoreload_tick()` is gone, so this can be removed. I could not edit the file myself. (Maybe `pypewpew` can check the box that maintainers can edit PRs?)

```suggestion
```